src/app/app.component.html

```diff
@@ -1,358 +1,10 @@
-<!-- * * * * * * * * * * * * * * * * * * * * * * * * * * * * * * * -->
-<!-- * * * * * * * * * * * The content below * * * * * * * * * * * -->
-<!-- * * * * * * * * * * is only a placeholder * * * * * * * * * * -->
-<!-- * * * * * * * * * * and can be replaced. * * * * * * * * * * * -->
-<!-- * * * * * * * * * * * * * * * * * * * * * * * * * * * * * * * -->
-<!-- * * * * * * * * * Delete the template below * * * * * * * * * * -->
-<!-- * * * * * * * to get started with your project! * * * * * * * * -->
-<!-- * * * * * * * * * * * * * * * * * * * * * * * * * * * * * * * -->
+<ng-container *ngIf="isMenuVisible">
+  <app-sidenav>
+    <router-outlet></router-outlet>
+  </app-sidenav>
+</ng-container>
 
```

Comment:
Here you should also have an else block with only 
`<router-outlet></router-outlet>`
Otherwise you will not see anything when "isMenuVisible" is false